ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/InternalToCertificate.java

```diff
@@ -24,14 +24,25 @@
 import se.inera.intyg.common.support.facade.builder.CertificateBuilder;
 import se.inera.intyg.common.support.facade.model.Certificate;
 import se.inera.intyg.common.ts_bas.v7.model.converter.certificate.MetaDataGrundData;
+import se.inera.intyg.common.ts_bas.v7.model.converter.certificate.category.CategoryIntygetAvser;
+import se.inera.intyg.common.ts_bas.v7.model.converter.certificate.question.QuestionIntygetAvser;
 import se.inera.intyg.common.ts_bas.v7.model.internal.TsBasUtlatandeV7;
 
 @Component(value = "internalToCertificateTsBas")
 public class InternalToCertificate {
 
     public Certificate convert(TsBasUtlatandeV7 internalCertificate, CertificateTextProvider texts) {
+        int index = 0;
         return CertificateBuilder.create()
             .metadata(MetaDataGrundData.toCertificate(internalCertificate, texts))
+            .addElement(
+                CategoryIntygetAvser.toCertificate(index++, texts)
+            )
+            .addElement(
+                QuestionIntygetAvser.toCertificate(
```

Comment:
I believe it is better to try to keep this class as free as possible from logic. Maybe just pass the "IntygAvser" and let any null checks etc happen in the QuestionIntygetAvser. This removes the need to create test-cases for this particular class for handling null, which can be a bit messy when so much is going on. 

And I noticed further down that the null test is missing, so point proven ;-)